relay/README.md

```diff
@@ -84,91 +65,189 @@ Scene 2:
 - QmOne tries to connect via relaying, because it shares this transport with QmTwo.
 - A lively and prolonged dialogue ensues.
 
-
 ## Addressing
 
-`/p2p-circuit` multiaddrs don't carry any meaning of their own.
-They need to encapsulate a `/p2p` address, or
-be encapsulated in a `/p2p` address, or both.
+`/p2p-circuit` multiaddrs don't carry any meaning of their own. They need to encapsulate a `/p2p` address, or be encapsulated in a `/p2p` address, or both.
+
+As with all other multiaddrs, encapsulation of different protocols determines which metaphorical tubes to connect to each other.
+
+A `/p2p-circuit` circuit address, is formated following:
+
+`/p2p-circuit[<relay peer multiaddr>]<destination peer multiaddr>`
```

Comment:
Also, after @lgierth clarifications this should be `<relay peer multiaddr>/p2p-circuit/<destination peer multiaddr>`